docs/pages/setup/operations/ca-rotation.mdx

```diff
@@ -313,5 +325,13 @@ $ tctl get nodes --format=json | jq '.[] | {hostname: .spec.hostname, rotation:
 }
 ```
 
-If any of the nodes were lost and using the old cert authority, they should reconnect
-once you switch the control plane to the old cert authority.
+If connectivity to any of the nodes was lost during the rotation, this is likely
+because they were still and using the old cert authority. Connectivity to these
+nodes should be restored when the rollback completes and the old certificate
+authority is made active.
+
+## Further reading
+
+- How the
+  [Teleport Certificate Authority](../../architecture/authentication.mdx#authentication-in-teleport)
+  works
```

Comment:
```suggestion
  works.
```

Reply:
I didn't think we were putting periods at the end of unordered lists. @ptgott ?

Reply:
We don't have a style guide rule around this. In reviews, I've been making sure that the use of periods (or not) is consistent between list items in a particular list. Since there's only one item here, I think either use is fine. We could also make this a single sentence instead of a list.